Data/IntMap.hs

```diff
@@ -31,20 +32,6 @@
 -- >  import Data.IntMap (IntMap)
 -- >  import qualified Data.IntMap as IntMap
 --
--- The implementation is based on /big-endian patricia trees/.  This data
```

Comment:
I want you to cite his article; if you or @ekmett can find prior art, that would be great to cite as well.